misc/selenium/Makefile

```diff
@@ -0,0 +1,55 @@
+# This Makefile may help you figure out how to run the selenium grid
+# service required for the webdriver frontend tests to work.  It is in
+# no way well-written or stable, and you should only run it after you
+# have read it.
+#
+# Some more pointers:
+#
+# Download the standalone grid server from here:
+#
+#     http://selenium-release.storage.googleapis.com/index.html
+#
+# Run like this:
+#
+# $ java -jar selenium-server-standalone-2.45.0.jar -role hub
+# $ java -jar selenium-server-standalone-2.45.0.jar -role node -hub http://localhost:4444/grid/register
+#
+# (You may have to change the version number to what you downloaded.)
+#
+# Here is a slightly outdated tutorial with partially helpful further
+# instructions: https://code.google.com/p/selenium/wiki/Grid2.
+
+
+SELENIUM_HOST=localhost
+
+SELENIUM_HUB_PORT=4451
+# (default: 4444)
+
+SELENIUM_NODE_PORT=4452
+# (default: 5555)
+
+SELENIUM_ARGS=-jar selenium-server-standalone-2.45.0.jar -host $(SELENIUM_HOST)
+
+default:
+	cat Makefile
+	make kill start watch
+
+init:
+	curl http://selenium-release.storage.googleapis.com/2.45/selenium-server-standalone-2.45.0.jar > selenium-server-standalone-2.45.0.jar
+	mkdir -p log
+
+kill:
+	killall java > /dev/null 2>&1 || true
```

Comment:
they run java.  their lives can only improve after this.  ;-)

yes, i know, this is all not very sturdy.  would you like to patch it?

(@fhartwig also wants to add a flag to disable selenium tests; then at least it will be easy (or even the default) to get away from this mess entirely.)